run-qa-checks

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+openwisp-qa-check --skip-checkmigrations
+
+# test sphinx docs
+mkdir -p docs/source/_static
+make -C docs html
```

Comment:
Can you please replicate this? https://github.com/openwisp/openwisp-radius/blob/master/run-qa-checks#L13-L17 (ensuring it fails if you create a broken link on purpose).
It's a great way to ensure we don't publish broken docs!

Reply:
Done